Assets/Scripts/Achievements/Achievements.cs

```diff
@@ -26,15 +27,19 @@ public class Achievements : MonoBehaviour
         [SerializeField] private Transform achievementDisplayContainer;
```

Comment:
Had a think about this class and now that I was looking at how the managers are laid out, it may make sense to remove this as being a mono behaviour too, since there will be a separate class handling the achievement display in the book presumably, and just run the constructor directly in the `GameManager`. @BenKellySoftware do you have any thoughts?

Reply:
bump @BenKellySoftware?